.eslintrc.js

```diff
@@ -33,37 +70,63 @@ module.exports = {
       },
     },
     {
-      'files': ['*.test.js', '*.test.jsx'],
+      'files': ['test/**/*.test.js'],
       'rules': {
         'jest/expect-expect': 'off',
         'jest/no-conditional-expect': 'off',
         'jest/no-done-callback': 'off',
         'jest/no-standalone-expect': 'off',
         'jest/no-try-expect': 'off',
-        'no-unused-vars': 'off',
       },
     },
     {
-      'files': ['**/*.ts', '**/*.tsx'],
-      'env': {
-        'node': true,
+      'files': ['test/**/cloudant.v1.test.js'],
+      'rules': {
+        'prettier/prettier': 'off',
+        'object-shorthand': 'off',
       },
-      'extends': ['airbnb-typescript/base', 'prettier'],
+    },
+    // TypeScript specific rules
+    {
+      'files': ['**/*.ts'],
+      'extends': [
+        'airbnb-typescript/base',
+        'plugin:import/recommended',
+        'plugin:import/typescript',
+        'prettier',
```

Comment:
I can see a duplication for `plugin:import/recommended` and `prettier`. Is it possible to use these in a global `extends` array regardless of the file extension?

Reply:
I could move `plugin:import/recommended` onto project level: 0df7972
I also added an additional tool based on this: [plugin:import/recommended](https://www.npmjs.com/package/eslint-plugin-import#typescript)

But prettier just formats the code differently, if I import it on project level. So I would leave currently as-it-is now.